src/phoenix/server/api/queries.py

```diff
@@ -84,6 +85,8 @@
 from phoenix.server.api.types.UserApiKey import UserApiKey, to_gql_api_key
 from phoenix.server.api.types.UserRole import UserRole
 
+initialize_playground_clients()
```

Comment:
Can we add this once inside `create_app`?

Reply:
good question, I think probably yes